src/Microsoft.IdentityModel.Tokens/Encryption/SymmetricKeyWrapProvider.cs

```diff
@@ -224,7 +224,7 @@ public override byte[] UnwrapKey(byte[] keyBytes)
                 throw LogHelper.LogArgumentNullException(nameof(keyBytes));
 
             if (keyBytes.Length % 8 != 0)
-                throw LogHelper.LogExceptionMessage(new ArgumentException(LogHelper.FormatInvariant(LogMessages.IDX10664, keyBytes.Length << 3), nameof(keyBytes)));
+                throw LogHelper.LogExceptionMessage(new ArgumentException(LogHelper.FormatInvariant(LogMessages.IDX10664, LogHelper.MarkAsNonPII(keyBytes.Length << 3)), nameof(keyBytes)));
```

Comment:
nameof(keyBytes) can be NonPII too right?

Reply:
nameof(keyBytes) falls outside the FormatInvariant method and cannot be marked as NonPII